docs/source/tips.rst

```diff
@@ -18,3 +18,60 @@ If you see that compilations run everytime you run the same script, then the cac
 Please check that the directory is kept as is between multiple executions of the script.
 If your home directory is not suited to caching the kernels (e.g. in case that it uses NFS), change the kernel caching directory by setting the ``CUPY_CACHE_DIR`` environment variable to an appropriate path.
 See :ref:`cupy-overview` for more details.
+
+
+mnist example does not converge in CPU mode on Mac OS X
+-------------------------------------------------------
+
+Many users reported that mnist example does not work correctly on Mac OS X.
+We are suspecting it is caused by vecLib, that is a default BLAS library installed on Mac OS X.
+
+.. note::
+
+   Mac OS X is not officially supported.
+   I mean it is not tested continuously on our test server.
+
+We recommend to use other BLAS libraries such as `OpenBLAS <http://www.openblas.net/>`_.
+We empirically found that it fixes this problem.
+Please reinstall NumPy with other BLAS library.
+Here is a instruction to install NumPy with OpneBLAS using `Homebrew <http://brew.sh/>`_.
+
+::
+
+   $ brew tap homebrew/science
+   $ brew install openblas
+   $ brew install numpy --with-openblas
+
+If you want to install NumPy with pip, use `site.cfg <https://github.com/numpy/numpy/blob/master/site.cfg.example>`_ file.
+
+After that, you can check if NumPy uses OpenBLAS with ``numpy.show_config`` method.
+
+::
+
+   >>> import numpy
+   >>> numpy.show_config()
```

Comment:
Can you point out which part we should check? Do we have to check that all entries are replaced with openblas?